sdk/go/common/workspace/project_test.go

```diff
@@ -434,6 +456,122 @@ config:
 	assert.Equal(t, "us-west-1", getConfigValue(t, stack.Config, "aws:region"))
 }
 
+func TestUntypedProjectConfigValuesAreNotValidated(t *testing.T) {
```

Comment:
Can we have another test very much like this one but checking that "Undeclared" values are still ok. So have instanceSize in the stackYaml but don't declare it at all in the projectYaml

Reply:
Config values in a stack don't need a declaration in the project when:
 1. They are non-project namespaced values (i.e. `aws:<something>`)
 2. The project does not declare any config yet (for backward-compatibility: how things are used right now)

When the project does declare any config value _and_ the stack uses an underclared value, we show an error saying that the value should be declared. Do we want to get rid of this?

Reply:
Dropped 2. and now we are more chill about having stacks declare config values not in the project (see tests)

Reply:
Yeh speaking to Luke about this we think we should err on being lenient here for now, something to revisit in the future once there's a lot of uptake on this.